scripts/init_dev

```diff
@@ -38,10 +38,15 @@ sudo apt-get update -q
 # sudo apt-get -y upgrade
 
 # Install pip and couchdb (couchdb is not available via rosdep -- see below)
-echo "Install pip, couchdb and boostrapping tools"
-sudo apt-get install -y -q python-pip couchdb
+echo "Install pip, couchdb and bootstrapping tools"
+sudo apt-get install -y -q couchdb
+sudo easy_install pip
```

Comment:
@gordonbrander I was trying to re-run the install script after I suspected it had failed in a prior attempt (I ran prior headless). Rerunning it gave `ImportError: cannot import name IncompleteRead` error at the pip commands [[1](https://github.com/OpenAgInitiative/openag_brain/blob/master/scripts/init_dev#L44), [2](https://github.com/OpenAgInitiative/openag_brain/blob/master/scripts/init_dev#L86)] in the init_dev file (I commented one out and it failed on another). I found [this solution](http://stackoverflow.com/questions/27341064/how-do-i-fix-importerror-cannot-import-name-incompleteread) and was able to run the re-install.